index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+     <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>Portfolio</title>
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+        <link rel="stylesheet" href="style.css">
+
+     </head>
+     <body>
+       <header>
+         <div class="head-wrapper">
+           <nav class="nav-bar">
+             <ul>
+               <li class="logo"><a href="#">My Logo</a></li>
+            </ul>
+            <div class="menu">
+                   <div class="toogle-btn"></div>
+                   <div class="toogle-btn"></div>
+                   <div class="toogle-btn"></div>
+             </div>
+           </nav>
```

Comment:
- [ ] Great job I can see you improve your design by following the requirement of the previous review. However, you still need to work on the alignment of the logo and the hamburger, add the property `align-items: center` to the ` <div class="head-wrapper">` in css class. check down here :point_down: 

The template | Your design 
------------------| ---------------
![Screenshot from 2021-11-25 13-17-23](https://user-images.githubusercontent.com/65068771/143432228-775b7cc2-8b92-4d21-a1fe-e9093c9cc538.png) |  ![Screenshot from 2021-11-25 13-14-09](https://user-images.githubusercontent.com/65068771/143432261-aaac9351-878f-439a-83a5-5b600298857c.png)
- [ ] By default all `<a>` are underlined but according to the design the logo should not be underlined. To remove this property just give the  <a> tag property `text-decoration: none` in css.